From e009760c2f7763a20b085a53f60cdb83da799fe5 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 09:39:44 +0300
Subject: [PATCH 10/30] arm: dts: s32cc: Remove unused clock-frequency property

Remove unused clock-frequency property from UART nodes.

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/dts/s32cc.dtsi | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/dts/s32cc.dtsi b/arch/arm/dts/s32cc.dtsi
index 4c2564e243..25c4c117a6 100644
--- a/arch/arm/dts/s32cc.dtsi
+++ b/arch/arm/dts/s32cc.dtsi
@@ -200,7 +200,6 @@
 				 <&clks S32GEN1_SCMI_CLK_LINFLEX_XBAR>;
 			clock-names = "lin", "ipg";
 			device_type = "serial";
-			clock-frequency = <0>;	/* updated dynamically if 0 */
 		};
 
 		uart1: serial@401cc000 {
@@ -211,7 +210,6 @@
 				 <&clks S32GEN1_SCMI_CLK_LINFLEX_XBAR>;
 			clock-names = "lin", "ipg";
 			device_type = "serial";
-			clock-frequency = <0>;	/* updated dynamically if 0 */
 		};
 
 		spi0: spi@401d4000 {
@@ -309,7 +307,6 @@
 				 <&clks S32GEN1_SCMI_CLK_LINFLEX_XBAR>;
 			clock-names = "lin", "ipg";
 			device_type = "serial";
-			clock-frequency = <0>;	/* updated dynamically if 0 */
 		};
 
 		spi3: spi@402c8000 {
-- 
2.17.1

